### ISSUE TITLE
Assignment / note and written feedback not recovered

### ISSUE THREAD

**Issue by pmercier125** [jpahullo/moodle-tool_mergeusers#267]
After a merge, we noticed that, for an assignment, the file was recovered, the assignment was well marked as 'noted', but the grade was 0/20 (incorrect) and the teacher's written feedback were not recovered.

**Comment by jpahullo** [comment]
Hi!

Thanks for the feedback, Patricia.

Please, detail plugin version and Moodle version.

Thanks,

Jordi

**Comment by pmercier125** [comment]
Moodle 4.1.6
Plugin v 2021122901


Cordialement,
Patricia


***@***.***

***@***.***
Patricia MERCIER
Conseillère techno-pédagogique
Cheffe de projet Université Virtuelle
DE - Centre d'appui pédagogique

Université libre de Bruxelles
Campus du Solbosch - CP111/03
Avenue F.D. Roosevelt, 50 - 1050 Ixelles
https://portail.ulb.be/fr/utiliser-le-numerique<https://portail.ulb.be/fr/enseignement/creer-et-innover-dans-un-enseignement/utiliser-le-numerique>


***@***.***<https://forms.office.com/e/z1W19wRiAE>

De : Jordi Pujol-Ahulló ***@***.***>
Envoyé : mardi 23 avril 2024 15:09
À : jpahullo/moodle-tool_mergeusers ***@***.***>
Cc : MERCIER Patricia ***@***.***>; Author ***@***.***>
Objet : Re: [jpahullo/moodle-tool_mergeusers] Assignment / note and written feedback not recovered (Issue #267)


Hi!

Thanks for the feedback, Patricia.

Please, detail plugin version and Moodle version.

Thanks,

Jordi

-

**Comment by jpahullo** [comment]
Hi @pmercier125,

It would really be helpful if you can also attach here the configuration options from the plugin in your Moodle site. There are some settings that may interfere in the expected results. Some screenshots or similar with all the plugin settings will be sufficient.

We have this plugin also in our production site, and we did not observed this behaviour, at least till now.

Thank you in advance!

Jordi

**Comment by pmercier125** [comment]
Hi
configuration options from the plugin :
***@***.***
***@***.***



Cordialement,
Patricia


***@***.***

***@***.***
Patricia MERCIER
Conseillère techno-pédagogique
Cheffe de projet Université Virtuelle
DE - Centre d'appui pédagogique

Université libre de Bruxelles
Campus du Solbosch - CP111/03
Avenue F.D. Roosevelt, 50 - 1050 Ixelles
https://portail.ulb.be/fr/utiliser-le-numerique<https://portail.ulb.be/fr/enseignement/creer-et-innover-dans-un-enseignement/utiliser-le-numerique>


***@***.***<https://forms.office.com/e/z1W19wRiAE>

De : Jordi Pujol-Ahulló ***@***.***>
Envoyé : mercredi 24 avril 2024 09:17
À : jpahullo/moodle-tool_mergeusers ***@***.***>
Cc : MERCIER Patricia ***@***.***>; Mention ***@***.***>
Objet : Re: [jpahullo/moodle-tool_mergeusers] Assignment / note and written feedback not recovered (Issue #267)


Hi @pmercier125<https://github.com/pmercier125>,

It would really be helpful if you can also attach here the configuration options from the plugin in your Moodle site. There are some settings that may interfere in the expected results. Some screenshots or similar with all the plugin settings will be sufficient.

We have this plugin also in our production site, and we did not observed this behaviour, at least till now.

Thank you in advance!

Jordi

-

**Comment by jpahullo** [comment]
Hi @pmercier125,

Recently we have addressed some conscious testing and we cannot reproduce your case.

In addition, you are replying to the email and things do not get visible on the github issue. So, it is better you to write and copy&paste whatever directly into this github issue.

As we cannot reproduce this, I am closing it by now.

Feel free to add more detail and configuration as I requested before, if you do feel that the problem persists.

Thanks for the feedback,

Jordi
